addon/components/o-s-s/attribute-text.ts

```diff
@@ -0,0 +1,32 @@
+import { assert } from '@ember/debug';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+interface OSSAttributeTextArgs {
+  label: string;
+  value?: string;
+  copyable?: boolean;
+  tooltip?: string;
+}
+
+export default class OSSAttributeText extends Component<OSSAttributeTextArgs> {
+  @tracked hasError: boolean = false;
+  @tracked displayInitials: boolean = false;
+
+  constructor(owner: unknown, args: OSSAttributeTextArgs) {
+    super(owner, args);
+  }
+
+  get label(): string {
+    assert(`[component][OSS::Attribute::Text] label is required `, this.args.label);
+    return this.args.label;
+  }
+
+  get value(): string | undefined {
+    return this.args.value === undefined || this.args.value.length === 0 ? '-' : this.args.value;
+  }
+
+  get copyable(): boolean {
```

Comment:
@JulienVannier66 wasn't the copy button supposed to be displayed on hover of the componentrow ? 🤔

Reply:
Also copyable is supposed to be truthy by default, pretty sure your condition below does not match the requirements

Reply:
@Miexil Yes it is. But in some case, for the gender for example, when hover the line, the OSS::Copy should stay hidden

Reply:
Managed with the method used by @Miexil after taking a look at his PR